parsons/ngpvan/custom_fields.py

```diff
@@ -0,0 +1,73 @@
+from parsons.etl.table import Table
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class CustomFields():
+
+    def __init__(self, van_connection):
+
+        self.connection = van_connection
+
+    def get_custom_fields(self, field_type='contacts'):
+        """
+        Get custom fields.
+
+        `Args:`
+            field_type : str
+                Filter by custom field group type. Must be one of ``contacts`` or
+                ``contributions``.
```

Comment:
In light of this binary distinction, and the unlikeliness that it will change any time soon, should we consider making this be a boolean along the line of `get_custom_fields(self, contrib_fields=False)` and then add a line of code to parse that above line 26 below?

Reply:
I considered that and I think that it is more straightforward to the user to keep it this way. Additionally, I included a default argument of `contacts` so presumably the user won't even have to enter in a value most of the time. And, as you mention, this will possibly future proof for additional custom field types that are included.